app/controllers/versions_controller.rb

```diff
@@ -78,7 +78,7 @@ def which_significance_changed(current_tag, previous_tag)
   end
 
   def load_and_authorize_resource
-    @cocina_object = Dor::Services::Client.object(params[:item_id]).find
+    @cocina_object = maybe_load_cocina(params[:item_id])
     authorize! :manage_item, @cocina_object
```

Comment:
this method is in `application_controller` and loads the object but prevents an exception from being thrown if the cocina representation fails